packages/query/src/block-processor.ts

```diff
@@ -251,6 +252,26 @@ export class BlockProcessor implements BlockProcessorInterface {
       let metadata: Metadata | undefined;
       metadata = await this.querier.shieldedPool.assetMetadata(assetId);
 
+      // If the metadata is for a delegation token, customize its symbol.
+      if (metadata && assetPatterns.delegationTokenPattern.test(metadata.display)) {
+        // We can't trust the validator's self-described name, so use their validator ID.
```

Comment:
What field is their "self-described" name? Is `metadata.display` safe to use?

Reply:
I mean fields in the validator definition like the name, description, etc (like "Penumbra Labs CI 1"). The metadata.display field has the validator ID in it, but this is long and unwieldy.